### PR TITLE
Add OperationalState Global Attributes test to the CI

### PR DESCRIPTION
**Pull request by marktrayer**
Steps:
- regen all yaml tests
- re-build chip-tool, re-build all-clusters
- manually run OPSTATE_1_1 via chip-tool
- verify resulting logs

### CODE REVIEW

File: src/app/tests/suites/certification/Test_TC_OPSTATE_1_1.yaml [hunk 1]
```diff
@@ -32,7 +32,6 @@ tests:
                 value: nodeId
 
     - label: "TH reads the ClusterRevision attribute from the DUT"
-      PICS: OPSTATE.S.Afffd
```
Comment (bzbarsky-apple) [inline src/app/tests/suites/certification/Test_TC_OPSTATE_1_1.yaml:35]: Why these PICS removals?

Comment (marktrayer) [reply src/app/tests/suites/certification/Test_TC_OPSTATE_1_1.yaml:35]: The Global Attribute PICS have been blanket removed from everywhere as I believe they're not used.  See all of the cluster TPs and all of the <cluster>_1_1 YAML tests in the repo.  So I'm just following the pattern as instructed.

**Review by bzbarsky-apple — APPROVED** [review]

**Comment by woody-apple** [comment]
@cecille to be the second reviewer here.

**Review by cecille — APPROVED** [review]

**Comment by marktrayer** [comment]
@woody-apple I've merged master into my branch.  It's still flagging as "out of date" though, and the lock example is failing, although I haven't touched anything remotely related (I don't think).

**Review by rbultman — APPROVED** [review]

**Pull request merged**